server/superdesk/commands/clean_images.py

```diff
@@ -19,27 +19,27 @@ class CleanImages(superdesk.Command):
     This command will remove all the images from the system which are not referenced by content.
     Probably running db.repairDatabase() is needed in Mongo to shring the DB size.
     """
+
     def run(self):
         try:
             print('Starting image cleaning.')
             used_images = set()
 
-            archive_items = superdesk.get_resource_service('archive').get(None, {'type': 'picture'})
+            archive_items = superdesk.get_resource_service('archive').get_from_mongo(None, {'type': 'picture'})
```

Comment:
@amagdas  I don't see any specific/proprietary mongo syntax. Probably if we switch to other storage server we still can be used the same syntax. What is important here is that by this function you force to get data from  storage server and not from search server. 
But it is a minor issue so not needed to insist more on it.